chromium_src/chrome/browser/ui/webui/favicon_source.cc

```diff
@@ -26,7 +26,6 @@
 #include "chrome/browser/profiles/profile.h"
 #include "url/gurl.h"
 // Just dummy values to avoid build errors
-namespace {
 namespace extensions {
```

Comment:
I believe this should be:
```
namespace extensions {
namespace {
```

otherwise these classes may conflict with similar ones defined somewhere else.

Reply:
Updated in fec11b38ff